salesagility/SuiteCRM
# Fix issue #6111 where override files are written to but not read from in module builder

Merge mod string arrays for lang and override files before returning





## Description
When building modstrings for module builder merge in the _override lang file before returning modstrings





## Motivation and Context
Fixes issue with users being unable to rename panels in the module builder


## How To Test This
Create a module in module builder, edit the detailview layout and try to change the panels labels, see issue #6111


## Types of changes


### Final checklist
